docs/quickstart.rst

```diff
@@ -60,38 +23,27 @@ users will start running within their own green gevent thread.
 
 .. code-block:: python
 
-    wait_time = between(5, 9)
-
-Our class defines a ``wait_time`` function that will make the simulated users wait between 5 and 9 seconds after each task 
-is executed. 
-
```

Comment:
Why move this to the bottom of the class?

Reply:
Because it is less important then everything else. According to 6/8 voters it shouldnt even be mandatory ;) https://github.com/locustio/locust/issues/1308

Reply:
I disagree. Even though `wait_time` can be declared as a method, in this example it's effectively a mandatory configuration paramater to the user class, so I think it should go at the top.

Reply:
In general, I agree that parameters should be first, but in this case (a quickstart) I believe we should introduce things in order of importance and wait_time is not as important as task definitions.

Reply:
With the current version (#1308 is a different issue I think), I'd argue that `wait_time` is just as important as tasks since they are both mandatory. And it's a parameter. Also the fact that we further down reference the wait time before having introduced it (see my other comment).